### PR TITLE
Departmentcontactemails

### PR DESCRIPTION
**Pull request by trmccormick**
# Department Contact Emails
Created a new Model called emails that is setup as a polymorphic association to the department model. So we have the ability to add email address in the same fashion that we add phone numbers to the department contact list.

Created or Modified the following files
-  /app/assets/stylesheets/frontend/_department.scss 
-  /app/assets/stylesheets/frontend/_display-list.scss
-  /app/controllers/admin/departments_controller.rb
-  /app/models/department.rb
-  /app/models/email.rb
-  /app/views/admin/_email.html.erb
-  /app/views/admin/departments/_form.html.erb
-  /app/views/admin/departments/index.html.erb
-  /app/views/department/details.html.erb
-  /app/views/department/list.html.erb
-  /db/migrate/20210212165404_create_emails.rb
-  /db/schema.rb
-  /spec/factories/departments.rb
-  /spec/factories/emails.rb
-  /spec/features/departments_spec.rb
-  /spec/models/email_spec.rb

## Description
Generated new migration / model called email. Model is setup as a polymorphic association to the department model. Updated views and generated rspec tests.

## Related Issue
https://github.com/wvulibraries/library_directory/issues/45

## Motivation and Context
Requested feature addition.

## How Has This Been Tested?
Created Rspec tests and also did manual testing when changes were completed.

## Screenshots (if appropriate):

### CODE REVIEW

File: app/views/admin/_email.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<fieldset class="emails-form">
+  <legend> Emails </legend>
+
+  <div class="field full">
+    <%= form.label :address, class: 'required' %>
+    <%= form.text_field :address, placeholder: 'somebody@mail.wvu.edu' %>
```
Comment (ddavisgraphics) [inline app/views/admin/_email.html.erb:6]: person@mail.wvu.edu

File: app/views/admin/departments/index.html.erb [hunk 2]
```diff
@@ -38,6 +38,21 @@
         </li>
         <% end %>
 
+        <% if dept.emails.length %>
```
Comment (ddavisgraphics) [inline app/views/admin/departments/index.html.erb:41]: present?

File: app/views/department/details.html.erb [hunk 3]
```diff
@@ -27,6 +27,17 @@
             </div>
           <% end %>
         </div>
+
+        <div class="email">
+          <% @department.emails.each do |email| %>
+            <div class="email-item">
+              <span class="fas fa-envelope"></span>
+              <a href="mailto:<%= email.address %>">
```
Comment (ddavisgraphics) [inline app/views/department/details.html.erb:35]: title="<%= email.address %>" 
OR 

title="<%= @department.name %> email address <%= @email.address %>"

**Review by ddavisgraphics — CHANGES_REQUESTED** [review]
Changes required.

File: app/views/admin/_email.html.erb (hunk 1 above)
Comment (trmccormick) [reply app/views/admin/_email.html.erb]: made adjustements

File: app/views/admin/departments/index.html.erb (hunk 2 above)
Comment (trmccormick) [reply app/views/admin/departments/index.html.erb]: made changes

**Pull request merged**